### ISSUE TITLE
[BUG] GitHub Actions

### ISSUE THREAD

**Issue by erictuvesson** [stackrox/kube-linter#152]
**System info:**
 - OS: Github

**Describe the bug**
Unable to run Github actions on any of my Kubernetes files.

I am guessing that it doesn't support having the yaml files in the root directory?
Does it scan all sub-directories though?

**To Reproduce**
In my Github repository I have a lot of Kubernetes files in the root folder while I have project specific ones 
in folders.

**Sample YAML input**
```yaml
# .github/workflows/kube-linter.yaml
name: Kube Linter

on: push

jobs:
  linter:
    name: Run Kube Linter
    runs-on: ubuntu-latest
    steps:
      - name: Scan yamls
        id: kube-lint-scan
        uses: stackrox/kube-linter-action@v1
        with:
          directory: .
```

Can't find any objects.

Output:
```log
Run stackrox/kube-linter-action@v1
  with:
    directory: .
Warning: no valid objects found.
```

Trying with config file. (where I tested having a config file named `kube-linter.yaml`, `.kube-linter.yaml` and a few more. Just whatever I do it removes the extension in the error and says it can't find it.

Output:
```log
Run stackrox/kube-linter-action@v1
  with:
    directory: yamls
    config: .kube-linter.yaml
Error: failed to load config: reading file: Config File ".kube-linter" Not Found in "[/home/runner/work/kubernetes-setup/kubernetes-setup]"
Error: Process completed with exit code 1.
```

**Expected behavior**
That it would find my files.

**Additional context**
I don't think the documentation of how it works is that well defined.
https://github.com/stackrox/kube-linter/tree/main/docs#kubelinter-github-action

I know there is an example in the actions repository, but that doesn't really help in my case.
https://github.com/stackrox/kube-linter-action

**Comment by neilcar** [comment]
@erictuvesson I believe you're missing a checkout action so your yaml isn't there to scan.  Try this:

```
# .github/workflows/kube-linter.yaml
name: Kube Linter

on: push

jobs:
  linter:
    name: Run Kube Linter
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Scan yamls
        id: kube-lint-scan
        uses: stackrox/kube-linter-action@v1
        with:
          directory: .
```

**Comment by erictuvesson** [comment]
🤦‍♂️ 

Thanks!

**Comment by neilcar** [comment]
When I tested this, I did the exact same thing...totally understandable.
